Phonebook/Phonebook/phoneBook.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+typedef struct
+{
+    char number[20];
+    char name[20];
+}Phonebook;
```

Comment:
```suggestion
} Phonebook;
```
И раз структура в .h-нике, ей тоже по-хорошему нужен комментарий